package.json

```diff
@@ -4,20 +4,23 @@
   "version": "0.0.0",
   "devDependencies": {
     "autoprefixer-core": "^5.2.1",
-    "babel-core": "6.26.0",
-    "babel-loader": "^6.2.4",
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
     "babel-plugin-transform-flow-strip-types": "^6.22.0",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-plugin-transform-runtime": "^6.23.0",
     "babel-preset-env": "^1.6.1",
-    "babel-preset-es2015": "^6.9.0",
+    "babel-preset-es2015": "^6.24.1",
```

Comment:
es2015 is deprecated - https://twitter.com/housecor/status/935551724911251461

Reply:
This was needed for the react-switch-button. Have removed.